lib/ddtrace/configuration/settings.rb

```diff
@@ -55,6 +56,18 @@ class Settings
         end
       end
 
+      settings :debug do
+        option :health_metrics do |o|
+          o.default do
+            Datadog::Debug::Health::Metrics.new(
+              enabled: env_to_bool(Datadog::Ext::Debug::Health::Metrics::ENV_ENABLED, true)
```

Comment:
we should default to `false`, just for initial release, then we'll enable by default later after some initial beta testing.

Reply:
Good idea, I'll change that.